crowbar_framework/config/locales/crowbar/en.yml

```diff
@@ -594,45 +594,39 @@ en:
         restore_chef: 'Restoring Chef server'
         restore_database: 'Restoring Crowbar database'
         description: |
-          Lorem Ipsum is simply dummy text of the printing and typesetting
-          industry. Lorem Ipsum has been the industry's standard dummy text ever
-          since the 1500s, when an unknown printer took a galley of type and
-          scrambled it to make a type specimen book.
+          The upgrade data was uploaded successfully. Start the restore process by clicking
+          'Restore'. This process can take a long time. It imports all required data from the old
+          SUSE OpenStack Cloud 5 Administration server from the upgrade data archive and runs the
+          install script in the background.
       repos:
         continue: 'Continue upgrade process'
         recheck: 'Recheck repositories'
-        header: 'Missing Repositories'
+        header: 'Verify Add On Product Repositories'
         description: |
-          The software repository for "SUSE Linux Enterprise High Availability
-          Extension" and "SUSE Linux Enterprise Storage" are missing. Please
-          configure these by hand and click recheck below when the repositories
-          are available.
+          This step verifies that the software repositories for
+          "SUSE Linux Enterprise High Availability Extension" and "SUSE Enterprise Storage" are
+          available if your setup requires them.
         ceph_repo: 'Ceph Repositories'
         ha_repo: 'HA Repositories'
       services:
-        continue: 'Continue upgrade process'
+        continue: 'Stop OpenStack Services'
         header: 'Stop OpenStack Services'
         description: |
-          Lorem Ipsum is simply dummy text of the printing and typesetting
-          industry. Lorem Ipsum has been the industry's standard dummy text ever
-          since the 1500s, when an unknown printer took a galley of type and
-          scrambled it to make a type specimen book.
+          After continuing the Upgrade by pressing 'Stop OpenStack Services' all services on the
+          client nodes will be stopped. This is the last possibility to access any OpenStack
+          Service. If not done yet, please shut down all running VM instances in your cloud and
+          backup anything what you need to use OpenStack APIs.
```

Comment:
Maybe instead of 'anything what you need' use 'everything you need'.
If not 'everything' then at least 'what' should be swapped for 'that'.